event/register_oracle_event.go

```diff
@@ -46,11 +58,114 @@ func (e RegisterOracleEvent) EventHandler(event ctypes.ResultEvent) error {
 		return err
 	}
 
-	err = sgx.VerifyRemoteReport(oracleRegistration.NodePubKeyRemoteReport, oracleRegistration.NodePubKey, *e.reactor.EnclaveInfo())
+	trustedBlockInfo := panacea.TrustedBlockInfo{
+		TrustedBlockHeight: oracleRegistration.TrustedBlockHeight,
+		TrustedBlockHash:   oracleRegistration.TrustedBlockHash,
+	}
+
+	queryClient, err := panacea.NewQueryClient(context.Background(), e.reactor.Config(), trustedBlockInfo)
+	if err != nil {
+		return err
+	}
+
+	txBuilder := panacea.NewTxBuilder(*queryClient)
+
+	voteOption := verifyReportAndGetVoteOption(oracleRegistration, e)
+
+	msgVoteOracleRegistration, err := makeOracleRegistrationVote(uniqueID, e.reactor.OracleAcc().GetAddress(), addressValue, voteOption, e.reactor.OraclePrivKey().Serialize(), oracleRegistration.NodePubKey)
 	if err != nil {
 		return err
 	}
 
-	// TODO: Executing Vote Txs
+	txBytes, err := generateTxBytes(msgVoteOracleRegistration, e.reactor.OracleAcc().GetPrivKey(), e.reactor.Config(), txBuilder)
+	if err != nil {
+		return err
+	}
+
+	if err := broadCastTx(e.reactor.GRPCClient(), txBytes); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// verifyReportAndGetVoteOption validates the RemoteReport and returns the voting result according to the verification result.
+func verifyReportAndGetVoteOption(oracleRegistration *types.OracleRegistration, e RegisterOracleEvent) types.VoteOption {
+	nodePubKeyHash := sha256.Sum256(oracleRegistration.NodePubKey)
+
+	if err := sgx.VerifyRemoteReport(oracleRegistration.NodePubKeyRemoteReport, nodePubKeyHash[:], *e.reactor.EnclaveInfo()); err != nil {
+		log.Infof("failed to verification report. uniqueID(%s), address(%s), err(%v)", oracleRegistration.UniqueId, oracleRegistration.Address, err)
```

Comment:
```suggestion
		log.Warnf("failed to verification report. uniqueID(%s), address(%s), err(%v)", oracleRegistration.UniqueId, oracleRegistration.Address, err)
```

Reply:
I understand. For now, I'll just change it to `Warnf`.